nss-cohort-37/Bangazon-Site-Red-Jaguars
# Jv like

# Description

User can like or dislike products. Products index view refactored to include product types filter.

Fixes #55 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git checkout jv-like
Like and dislike some products (you must be logged in)
Make sure product index view looks noice

# Checklist:
